chapters/ru/chapter8/certification.mdx

```diff
@@ -0,0 +1,33 @@
+# Получение сертификата о прохождении курса
+
+Процесс сертификации является полностью бесплатным.  
+* Для получения сертификата об окончании курса необходимо сдать 3 из 4 практических заданий до 1 сентября 2023 года.
+* Для получения сертификата с отличием об окончании курса необходимо сдать 4 из 4 практических заданий до 1 сентября 2023 года.
+
+Требования к каждому заданию приведены в соответствующих разделах:
+* [Раздел 4 Практическая работа](../chapter4/hands_on)
```

Comment:
In the unit itself, "hands on" is translated as "Практическое занятие". I don't mind either of the translations, but it might be better to stick to one of them.